### PR TITLE
Add LDAP configuration properties for ATC

### PR DESCRIPTION
**Pull request by JamesClonk**

### CODE REVIEW

**Comment by JamesClonk** [comment]
I've tested it in a self-built concourse dev-release, the LDAP feature of atc/dex seems to work out of the box. :+1:

File: jobs/atc/templates/ldap_ca_cert.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<%= p("ldap_auth.ca_cert", "") %>
```
Comment (vito) [inline jobs/atc/templates/ldap_ca_cert.erb]: I think this'll need to be `ldap_auth.ca_cert.certificate` since the type for the property is `certificate`.

Comment (JamesClonk) [reply jobs/atc/templates/ldap_ca_cert.erb]: @vito Hmm, I think it's fine like that. github_auth, cf_auth, etc. also only directly have `ca_cert`, I just copied their structure. Or is it wrong there?

Comment (vito) [reply jobs/atc/templates/ldap_ca_cert.erb]: @JamesClonk They were all actually broken - https://github.com/concourse/concourse/pull/2479 fixed them. :)

**Comment by JamesClonk** [comment]
@vito my bad, just saw the other merged PR (https://github.com/concourse/concourse/pull/2479).
Changed it accordingly. :smiley:

File: jobs/atc/templates/ldap_ca_cert.erb (hunk 1 above)
Comment (JamesClonk) [reply jobs/atc/templates/ldap_ca_cert.erb]: @vito changed it :+1:

**Comment by vito** [comment]
thanks!

**Pull request merged**